### ISSUE TITLE
What programs would you recommend i use to do this, i have ida, what tools do you use to get everything i would need to copy your method, i already use cs2-dumper for base offsets... do you use like cheat engine ? to get values like this "0x30"

### ISSUE THREAD

**Issue by TestMagicCode** [TKazer/CS2_External#133]
What programs would you recommend i use to do this, i have ida, what tools do you use to get everything i would need to copy your method, i already use cs2-dumper for base offsets... do you use like cheat engine ? to get values like this "0x30"

_Originally posted by @TestMagicCode in https://github.com/TKazer/CS2_External/issues/132#issuecomment-1806050951_

**Comment by pipi-ly** [comment]
Cheat Engine

**Comment by TestMagicCode** [comment]
> Cheat Engine

yes but what do you do to get the value, i have looked all over the place and tried many thing and nothing gives these values ?
